lib/tasks/deployment/20231125151610_set_case_contacts_as_active.rake

```
@@ -0,0 +1,15 @@
+namespace :after_party do
+  desc "Deployment task: set_case_contacts_as_active"
+  task set_case_contacts_as_active: :environment do
+    puts "Running deploy task 'set_case_contacts_as_active'"
+
+    CaseContact.all.each do |cc|
+      cc.update(status: "active", draft_case_ids: [cc.casa_case_id])
```

```suggestion
      cc.update!(status: "active", draft_case_ids: [cc.casa_case_id])
```

Let's not silently fail